src/main/java/com/synopsys/integration/alert/web/user/RoleController.java

```diff
@@ -73,6 +74,20 @@ public RoleController(ContentConverter contentConverter, ResponseFactory respons
         return responseFactory.createOkContentResponse(contentConverter.getJsonString(roleActions.getRoles()));
     }
 
+    @PutMapping
+    public ResponseEntity<String> updateRole(@RequestBody RolePermissionModel rolePermissionModel) {
+        if (!hasPermission(authorizationManager::hasCreatePermission)) {
```

Comment:
Should this be checking `create`? Do we have another permission type?

Reply:
Changed to write